lib/jekyll-feed/feed.xml

```diff
@@ -45,8 +45,10 @@
   {% endif %}
   {% for post in posts limit: 10 %}
     <entry{% if post.lang %}{{" "}}xml:lang="{{ post.lang }}"{% endif %}>
-      <title type="html">{{ post.title | smartify | strip_html | normalize_whitespace | xml_escape }}</title>
-      <link href="{{ post.url | absolute_url }}" rel="alternate" type="text/html" title="{{ post.title | xml_escape }}" />
+      {% assign post_title = post.title | smartify | strip_html | normalize_whitespace | xml_escape %}
+
+      <title type="html">{{ post_title }}</title>
+      <link href="{{ post.url | absolute_url }}" rel="alternate" type="text/html" title="{{ post_title }}" />
```

Comment:
I can see that the `href` attribute hasn't changed from the existing value on `master`.
But shouldn't that have an ` | xml_escape` as well..?

/cc @pathawks

Reply:
Would you like me to fix that as well? I can do that

Reply:
I know *you can*.., was expecting a confirmation from @pathawks though..

Reply:
Feel free to fix this in another issue. thanks.